bitbots_ros_control/cfg/bitbots_ros_control_params.cfg

```diff
@@ -10,24 +10,6 @@ from dynamic_reconfigure.parameter_generator_catkin import *
 gen = ParameterGenerator()
 #       Name       Type      Level    Description     Default Min   Max
 
-gen.add("read_position", bool_t, 1,
-        "Enable reading the current position values of the servos.")
-gen.add("read_velocity", bool_t, 1,
-        "Enable reading the current velocity values of the servos.")
-gen.add("read_effort", bool_t, 1,
-        "Enable reading the current effort values of the servos.")
-gen.add("read_volt_temp", bool_t, 1,
-        "Enable reading the current input voltages and temperature values of the servos.")
-gen.add("VT_update_rate", int_t, 1,
-        "How often the voltage and temperature should be read, i.e. putting 10 would mean every 10th time the position is read, VT is also read.", min=0, max=1000)
-gen.add("read_pressure", bool_t, 1,
-        "Enable reading the current pressure values of the foot")
-gen.add("read_imu", bool_t, 1,
```

Comment:
`read_pressure` and `read_imu` were removed and not added to the other cfg file. Is this on purpose?

Reply:
thats correct. This config options have been removed, since I think they are not very useful in the future and it was a bit strange before. Originally there was the mode to only read imu or only read pressure, and on top of that also these config values which were basically never used. The only imu/pressure modes are nice to test the sensors, but using the robot (meaning the servos) without the sensors makes relatively little sense, since it will break on other points (mainly HCM). Also it makes not much sense to change what is read while ros_control is running. I could introduce an only servo mode, if you want to test servos without sensors.
I also saw that I forgot to remove the read imu/pressure values from the config and fixed this in the next commit.